PicaSWB.js

```diff
@@ -39,6 +48,16 @@ var nachnameMapping = {
 };
 var nameMapping = {
```

Comment:
ja, 0999 Feld kann frei belegt werden. Beim Abspeichern kann 0999 je nach erfolgter bzw. nicht erfolgter Autorenverknüpfung gelöscht bzw. stehen gelassen werden. Mit dem o.g. Editor kann das Trefferergebnis  - PPN - in ognd bei richtiger Zuordnung zum Verfasser übernommen werden. Dein Lookup über das Web teste ich heute noch. Wenn Quick-Copy nicht geht, kann man alle Daten zunächst in eine Textdatei exportieren und sukzessive abarbeiten.